Sources/TuistKit/Commands/InspectCommand.swift

```diff
@@ -0,0 +1,46 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+import TuistSupport
+
+// MARK: - InspectCommand
+
+struct InspectCommand: ParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(commandName: "inspect",
+                             abstract: "Inspects a target from the dependency graph and prints information about it.")
+    }
+
+    // MARK: - Options
+
+    @OptionGroup()
+    var options: DocCommand.Options
+
+    // MARK: - Run
+
+    func run() throws {
+        let absolutePath: AbsolutePath
+        if let path = options.path {
+            absolutePath = AbsolutePath(path, relativeTo: FileHandler.shared.currentPath)
+        } else {
+            absolutePath = FileHandler.shared.currentPath
+        }
```

Comment:
It is not related to this PR but I would like to mention it:
1. Defining path is duplicated across commands implementations.
2. Some commands like: `Dependencies`, `Up`, `Generate` contain this logic into theirs services (`*Service`). Some others like `Doc`, `Test` and this one have this logic here (`*Command`).

Reply:
ad.1 Can we somehow abstract this logic? 
ad.2 Should we be consistent or this case is not important.